### PR TITLE
Replace Markdown with YAML issue templates

### PR DESCRIPTION
**Pull request by SwoopX**
As the title says, we're replacing the templates here.

Benefits are:
- Visually more appealing
- Fields become mandatory. Issue cannot be submitted without content.
- Notes/guidance/supplementary information is now directly presented and really hard to ignore 🙂 

In future, further validation/completeness checks might be enforced via Github actions to ensure high issue quality.

### CODE REVIEW

**Comment by Mimiix** [comment]
Thanks a bunch! I'll review this tommorow

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      label: Environment
+      description: |
+        The deCONZ and firmware versions are found in:
+        - Phoscon App: Menu -> Settings -> Gateway, OR
+        - deCONZ GUI: Menu > Help > About deCONZ.
+      value: |
+           - Host system: (Raspberry Pi / PC / NAS / MAC)
+           - Running method: (Raspbian / Ubuntu / Home Assistent deCONZ Add-on / Marthoc Docker container / Windows / Virtual Machine)
+           - Firmware version: (26xxyy00)
+           - deCONZ version: (2.xx.yy)
```
Comment (Mimiix) [inline .github/ISSUE_TEMPLATE/bug_report.yml:72]: Perhaps change this to : deCONZ version (not Home assistant Addon version!):

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+           - Device: (ConBee I / ConBee II / RaspBee I / RaspBee II)
```
Comment (Mimiix) [inline .github/ISSUE_TEMPLATE/bug_report.yml:73]: Add ConBee III already

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+           - Do you use an USB extension cable: (yes / no) -- only relevant for ConBee I/II
```
Comment (Mimiix) [inline .github/ISSUE_TEMPLATE/bug_report.yml:74]: Ditto Conbee 3

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+           - Is there any other USB or serial devices connected to the host system? If so: Which?
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: deCONZ Logs
+      description: |
+        Debug logs can help to investigate certain problems like device pairing and communication issues. Logging can be enabled on the command line as described in: https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/deCONZ-debug-flags
+        Secondly, for versions above 2.05.87: In the deCONZ GUI: click Help > debug view. Output of APS,ZDP, ZCL and INFO can be handy.
```
Comment (Mimiix) [inline .github/ISSUE_TEMPLATE/bug_report.yml:83]: I generally like to see:
APS
APS_L2
ERROR
ERROR_L2
INFO
INFO_L2

I am not sure on ZCL / ZDP.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 2 above)
Comment (SwoopX) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Yup

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 3 above)
Comment (SwoopX) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Yup

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 4 above)
Comment (SwoopX) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Agree. Made it a bit more cautious though.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (SwoopX) [reply .github/ISSUE_TEMPLATE/bug_report.yml:71]: Then it is probably better to throw it out completely (Home Assistent deCONZ Add-on I mean)?

Comment (Mimiix) [reply .github/ISSUE_TEMPLATE/bug_report.yml:71]: Not really. People tend to put in the addon version which is a different number then the deconz version. The addon uses different versioning numbers.

Comment (SwoopX) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Ah, now I got what you mean :) Done

**Pull request merged**